pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -531,6 +531,14 @@ containing the generator terminals used as angle reference in the load flow calc
 There is one Terminal created/added in the extension for each calculated Synchronous Component.
 Works only when `referenceBusSelectionMode` is set to `GENERATOR_REFERENCE_PRIORITY`.
 
+**voltageTargetPriorities**  
+This parameter allows to configure the priority order between voltage targets. 
+By default, the order is `["GENERATOR", "TRANSFORMER", "SHUNT"]`.
```

Comment:
should add a note that GENERATOR includes indistinctively generators, static var compensators, and VSC HVDC converters.